packages/discord.js/src/structures/GuildMember.js

```diff
@@ -396,6 +405,11 @@ class GuildMember extends Base {
    * guildMember.disableCommunicationUntil(Date.now() + (5 * 60 * 1000), 'They deserved it')
    *   .then(console.log)
    *   .catch(console.error);
+   * @example
+   * // Remove the timeout of a guild member
+   * guildMember.disableCommunicationUntil(null)
+   *   .then(member => `Removed timeout for ${member.displayName}`)
```

Comment:
Did you forgot to log this?

Reply:
That was supposed to happen yeah, my bad. I can't change it for the next couple of hours though as I'm out of town.